src/components/ProductCard.jsx

```diff
@@ -49,10 +58,30 @@ export default function ProductCard({ product }) {
       </div>
       <hr />
       <div className="card-footer flex-around flex-grow-1">
-        <button className="btn btn-atc btn-cta btn-md txt-bold txt-reg w-100">
+        {state["cart"].some((item) => item.id === product._id) ? (
+          <button className="btn btn-wish btn-md txt-bold txt-reg w-100">
+            <i className="fas fa-shopping-cart"></i>
+            go to cart
+          </button>
+        ) : state["wishlist"].some((item) => item.id === product._id) ? (
+          <button className="btn btn-wish btn-md txt-bold txt-reg w-100">
+            <i className="fas fa-shopping-cart"></i>
+            move to cart
+          </button>
+        ) : (
+          <button className="btn btn-atc btn-cta btn-md txt-bold txt-reg w-100">
+            <i className="fas fa-shopping-cart"></i>
+            add to cart
+          </button>
+        )}
+        {/* <button className="btn btn-atc btn-cta btn-md txt-bold txt-reg w-100">
           <i className="fas fa-shopping-cart"></i>
-          Add to cart
+          add to cart
         </button>
+        <button className="btn btn-wish btn-md txt-bold txt-reg w-100">
```

Comment:
unwanted comments should be removed

Reply:
Yes I was testing the product cards and forgot to cleanup the comments! Thanks for pointing it out!